openmdao/docs/feature_reference/core_features/defining_components/metadata.rst

```diff
@@ -0,0 +1,82 @@
+.. _component_metadata:
+
+********************************************
+Component metadata (arguments to components)
+********************************************
+
+The primary job of a component, whether explicit or implicit, is to define inputs and outputs,
+as well as the mapping that computes the outputs given the inputs.
+Often, however, there are incidental parameters that affect the behavior of the component
+but are not considered input variables in the sense of being computed as an output of another component.
+
+For convenience, OpenMDAO provides a way of declaring these parameters, which are referred to as *metadata*.
+They are declared in the 'initialize' method of the user's component,
+and the values are typically passed in upon instantiation of the component.
+Alternatively, the values can be set at any time inside or outside of the component.
```

Comment:
what does "inside or outside" the component mean? This sentence could be made more clear

Reply:
Rephrased